wiki/Beatmap/Pattern/Mania/Anchors/en.md

```diff
@@ -0,0 +1,13 @@
+---
+stub: true
+tags:
+  - anchors
+---
+
+# Anchors
+
+**Anchors**, usually referring to a byproduct of [streams](Mania/Streams) or sometimes [chordjacks](Mania/Jacks/en.md#chordjacks), are when a high concentration of notes in one column appear in consecutive snap intervals, typically in 1/2nd snap intervals. The name **anchor** comes from the motion the pattern creates, in which your finger is essentially anchored to that key due to the consecutive, intervallic presses. 
```

Comment:
I think the phrasing is a little off still, though I agree that it's not the best at the moment. I think the issue with yours is that it implies that anchors are an isolated pattern, rather than something that can be a byproduct of streams or chordjacks as well. What do you think of:
```suggestion
**Anchors**, usually referring to a byproduct of [streams](Mania/Streams) or sometimes [chordjacks](Mania/Jacks/en.md#chordjacks), are a high concentration of notes in one column that appear consecutively within the same snap interval, typically in 1/2nd snap intervals. The name **anchor** comes from the motion the pattern creates, in which your finger is essentially anchored to that key due to the consecutive, intervallic presses. 
```